docs/README.md

```diff
@@ -48,7 +48,10 @@ In layman's terms, BTCPay Server is a self-hosted and automated invoicing system
 
 ### How is it different
 
-BTCPay is code, not a company. There is no third-party between a merchant and a customer. Merchant is always in full control of the funds. There are no processing or subscription fees. BTCPay Server is free to use and completely open-source, so developers or security auditors can always inspect the quality of the code.
+BTCPay Server is free, open-source, self-hostable software. 
+BTCPay Server is not a company. To use BTCPay Server, you can register an account on the [official demo](https://mainnet.demo.btcpayserver.org/Account/Login), [deploy](./Deployment.md) your own instance, or use a [third-party host](./ThirdPartyHosting.md).
```

Comment:
Not sure if we should promote "use" with the demo instance. We should be more explicit here imho.

Reply:
I get what you're saying.
Yet it's stated as a demo.

Maybe the wording as follows could be better ?

```suggestion
BTCPay Server is not a company. To use BTCPay Server, [deploy](./Deployment.md) your own instance, or use a [third-party host](./ThirdPartyHosting.md).

 To explore what BTCPay Server has to offer, head over to the [official demo](https://mainnet.demo.btcpayserver.org/Account/Login).
```

Reply:
I think this is better.